### PR TITLE
Fixed all french contents and some typos

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Some labels need to be updated and translated.
I also fixed some typos.

I think a lot of french contents are from ``backbee/BackBee`` so I will check also the core php project.

Please review :)

### CODE REVIEW

File: src/tb/apps/contribution/templates/index.twig [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
         <p>
             <button type="button" class="btn btn-simple btn-xs">
-                <i class="fa fa-search"></i> Tous les contenus
+                <i class="fa fa-search"></i> Contents
             </button>
         </p>
         <p>
             <button type="button" class="btn btn-simple btn-xs">
-                <i class="fa fa-inbox"></i> Mediathèque
+                <i class="fa fa-inbox"></i> Media
```
Comment (fkroockmann) [inline src/tb/apps/contribution/templates/index.twig]: Media library better no ?

File: src/tb/apps/contribution/templates/index.twig [hunk 2]
```diff
@@ -46,14 +46,14 @@
     <div class="col-bb5-12">
         <p>
             <button id="global-save" type="button" class="btn btn-default-grey btn-sm btn-block">
-                <i class="fa fa-floppy-o"></i> Enregistrer
+                <i class="fa fa-floppy-o"></i> Register
```
Comment (fkroockmann) [inline src/tb/apps/contribution/templates/index.twig]: Save better no ?

File: src/tb/apps/contribution/templates/index.twig (hunk 1 above)
Comment (mickaelandrieu) [inline src/tb/apps/contribution/templates/index.twig]: Ok, i update

File: src/tb/apps/contribution/templates/index.twig (hunk 2 above)
Comment (mickaelandrieu) [inline src/tb/apps/contribution/templates/index.twig]: Updated too

File: src/tb/apps/page/templates/contribution.index.twig [hunk 3]
```diff
@@ -15,49 +15,49 @@
         </p>
         <p class="position-bottom">
             Scheduling :
-            <span id="contribution-scheduling-state">Aucune</span>
+            <span id="contribution-scheduling-state">No one</span>
             <button id="contribution-scheduling-btn" type="button" class="btn btn-transparent btn-xs btn-toggle btn-show">
-                <i class="fa fa-angle-down"></i> Programmer la publication / l'archivage
+                <i class="fa fa-angle-down"></i> Set the publishing / archiving
```
Comment (ndufreche) [inline src/tb/apps/page/templates/contribution.index.twig]: Just "publishing/archiving" is enough i think.

File: src/tb/component/contentselector/templates/layout.tpl [hunk 4]
```diff
@@ -28,14 +28,14 @@
           <div class="row">
             <div class="col-bb5-x">
               <div class="row form-group"><span class="col-md
-  col-sm-6"><label class="sr-only" for="form10">Titre</label><input id="form10" type="text" placeholder="Titre" class="form-control input-xs"></span> <span class="col-bb5-x pull-right bb5-query-results">Article - 593 éléments</span></div>
+  col-sm-6"><label class="sr-only" for="form10">Title</label><input id="form10" type="text" placeholder="Titre" class="form-control input-xs"></span> <span class="col-bb5-x pull-right bb5-query-results">Article - 593 elements</span></div>
```
Comment (ndufreche) [inline src/tb/component/contentselector/templates/layout.tpl]: You forgot le placeholder

File: src/tb/apps/page/templates/contribution.index.twig (hunk 3 above)
Comment (mickaelandrieu) [inline src/tb/apps/page/templates/contribution.index.twig]: I'm not sure user will understand without the "Set" :/

Comment (fkroockmann) [inline src/tb/apps/page/templates/contribution.index.twig]: I agree with @ndufreche

**Comment by ndufreche** [comment]
Nice job, thanks for this improvement !
:+1:

**Comment by fkroockmann** [comment]
Thanks :+1:

File: src/tb/apps/page/templates/contribution.index.twig (hunk 3 above)
Comment (ndufreche) [inline src/tb/apps/page/templates/contribution.index.twig]: I don't know @mickaelandrieu is just an opinion

File: src/tb/component/contentselector/templates/layout.tpl (hunk 4 above)
Comment (mickaelandrieu) [inline src/tb/component/contentselector/templates/layout.tpl]: nice catch, updated ;)

**Comment by mickaelandrieu** [comment]
I squashed my commits. This is now ready for merge: thank you all for review :+1:

**Pull request merged**